index.html

```diff
@@ -0,0 +1,250 @@
+   <h4 class="featured-heading">Barr Moses</h4>
+   <p class="featured-para"> <i>Co-founder & CEO at Monte Carlo</i></p>
+   <span class="featured-span">Barr is co-founder of data reliability company Monte Carlo.
+    Barr is co-founder of data reliability company Monte Carlo.
+   </span> 
+  </div>
+    </section>
+    <section class="featured-speaker">
+        <div class="image-of-background">
+      <img class="featured-speakers" src="./images/speaker2.png" alt="speaker1">
+        </div>
+        <div class="fetured-text">
+         <h4 class="featured-heading">Changpeng Zhao</h4>
+         <p class="featured-para"><i>Co-founder & CEO at Binance</i></p>
+         <span class="featured-span">In 2020, Bloomberg recognised Changpeng as one of the
+             most influential people of the year, and as someone who has changed the global business landscape. 
+         </span> 
+        </div>
+          </section>
+        <div class="seemore-div">
+          <button class="see-more">See More</button>
+        </div>
+          <div class="dynamic">
+
+          </div>
+</article>
+<article id="featured-desktop">
+    <h3 class="featuredh">Featured Speakers</h3>
+    <hr class="hr1"/>
+    <section class="featured-speaker-desktop">
+          
+            <div class="featured-table">
+                <table>
+                    <tr>
+                        <td><div class="image-of-background">
+                            <img class="featured-speakers" src="./images/speaker1.jpg" alt="speaker1">
+                        </div></td>
+                        <td><h4 class="featured-heading-">Barr Moses</h4>
+                            <p class="featured-para"> <i>Co-founder & CEO at Monte Carlo</i></p>
+                            <span class="featured-span">Barr is co-founder of data reliability company Monte Carlo.
+                             Barr is co-founder of data reliability company Monte Carlo.
+                            </span> </td>
+                        <td><div class="image-of-background">
+                            <img class="featured-speakers" src="./images/speaker2.png" alt="speaker1">
+                        </div>
+                        </td>
+                    <td><h4 class="featured-heading">Changpeng Zhao</h4>
+                        <p class="featured-para"><i>Co-founder & CEO at Binance</i></p>
+                        <span class="featured-span">In 2020, Bloomberg recognised Changpeng as one of the
+                            most influential people of the year, and as someone who has changed the global business landscape. 
+                        </span> </td>
+                    </tr>
+                </table>
+            </div>
+            <div class="seemore-div-desktop">
+                <button class="see-more-desktop">See More</button>
+              </div>
+                <div class="dynamic-desktop">
+      
+                </div>
+    </section>
+</article>
```

Comment:
- [ ] Please take your time and understand well how semantic html works and structured. I have gave you a clear example that shows you how you could implement structure your html elements tags to make it more accessible, but seemed that you've just changed a small part.   It is very important for your project to be more semantic because with it, it is more accessible.  Revisit the [example ](https://gist.github.com/ishimwezachee/27635e614feba10b49df5b172915e8aa)I gave and refer to it.
```
<header>
     <h1> </h1>
<header>
<main>
     <section>
               <h3></h3>
              <article>
               </article>
      </section>
        <section>
               <h3></h3>
              <article>
               </article>
      </section>
</main>

<footer>
</footer>

```

Reply:
Hi Zachee, As discussed on zoom call I have replaced article tags with section tags while on zoom call screen share with you as per your instructions and added the code in main tag. Sending the code for a review again this is my fifth review for this capstone project please approve.

Reply:
@baqar-abbas Always consider tagging the Code Reviewer if you have some questions by using @ before their GitHub Handle so they will receive a notification. 

In short, I see you have gone through many reviews these things happen because the student doesn't do the project exactly in accordance with the guidelines and the Code Reviewer has not approved it. Consider always rechecking your project with guidelines that are all met.